pkg/ctl/utils/describe_stacks.go

```diff
@@ -29,20 +53,23 @@ func describeStacksCmd(cmd *cmdutils.Cmd) {
 		fs.BoolVar(&all, "all", false, "include deleted stacks")
 		fs.BoolVar(&events, "events", false, "include stack events")
 		fs.BoolVar(&trail, "trail", false, "lookup CloudTrail events for the cluster")
+		fs.StringVarP(&output, "output", "o", "", "specifies the output formats (valid option: json and yaml)")
 		cmdutils.AddTimeoutFlag(fs, &cmd.ProviderConfig.WaitTimeout)
 	})
 
 	cmdutils.AddCommonFlagsForAWS(cmd.FlagSetGroup, &cmd.ProviderConfig, false)
 }
 
-func doDescribeStacksCmd(cmd *cmdutils.Cmd, all, events, trail bool) error {
+func doDescribeStacksCmd(cmd *cmdutils.Cmd, all, events, trail bool, printer printers.OutputPrinter, output printers.Type) error {
```

Comment:
Since output is no longer used:
```suggestion
func doDescribeStacksCmd(cmd *cmdutils.Cmd, all, events, trail bool, printer printers.OutputPrinter) error {
```